analytics-integrations/taplytics/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply from: rootProject.file('gradle/integration.gradle')
+
+dependencies {
+  compile files('libs/taplytics.jar')
```

Comment:
Could you rename this file to be `Taplytics-2.7.1.jar` or whatever the latest version is so it's easier to identify.

Also, is a Maven dependency available?